test/fetch-resources-for-products-test.js

```diff
@@ -0,0 +1,64 @@
+import assert from 'assert';
+import fetchResourcesForProducts from '../src/fetch-resources-for-products';
+
+suite('fetch-resources-for-product-test', () => {
+  test('it fetches all pages of images and for a single product', () => {
+    const imagesArray = ['images'];
+    const variantsArray = ['variants'];
+    const fixture = {
+      images: imagesArray,
```

Comment:
Where are we using `fixture.images` and `.variants`? I'm only seeing them access via `.attrs`.

Reply:
because the test shouldn't guarantee implementation. If this goes through anything that has expectations of model shape, it will expect the top level proxy fields. `GraphModel` has an internall `attrs` hash, and top level fields may be proxies, or overrides.